load_postgres_data.py

```diff
@@ -0,0 +1,74 @@
+import os
+import sys
+import time
+import subprocess
+import signal
+import tempfile
+import getpass
+
+conf = {
+    'staging': {
+        'port': 5432
+    },
+    'production': {
+        'port': 5433
+    }
+}
+
+
+def run_pg_cmd_on_guest(guest_cmd, password):
+    return subprocess.run([
+        ("docker-compose exec -e PGPASSWORD={password} "
+         "postgres bash -c '{guest_cmd}'").format(
+             password=password, guest_cmd=guest_cmd)
+    ], shell=True, check=True)
+
+
+def main(dbs, cloud_sql_proxy_port, cloud_sql_db_user, cloud_sql_db_password,
+         api_db_docker_user):
+    dumps_base_path = '/db_dumps'
+    for db in dbs:
+        file_name = '{db}.dump'.format(db=db)
+        dump_path = os.path.join(dumps_base_path, file_name)
+
+        guest_cmd = (
+            "pg_dump --host cloud_sql_proxy --port {port} --username={user} "
+            "-Fc --dbname={db} > {out_path}"
+        ).format(port=cloud_sql_proxy_port, user=cloud_sql_db_user,
+                 db=db, out_path=dump_path)
+        print("Dumping database: {cmd}".format(cmd=guest_cmd))
+        run_pg_cmd_on_guest(guest_cmd, cloud_sql_db_password)
+        print("Done!")
+
+        guest_cmd = "createdb -U {user} {db}".format(
+            user=api_db_docker_user, db=db)
+        print("Creating database: {cmd}".format(cmd=guest_cmd))
```

Comment:
We don't use `format()` in our Python projects. Alternatives: f-strings or `%`.

Reply:
The reason is because I wanted this script to be as much compatible as possible so that whoever wanna run this doesn't need to run a specific version of python3, that's why I avoided using `f-strings`, but I'm ok with `%` 👍